scripts/deploy-v2.js

```diff
@@ -92,7 +92,7 @@ const networks = {
 
 
 async function main() {
-
+  console.log({ host: process.env.NEXT_DB_DATABASE, ...process.env })
```

Comment:
use `console.table`

Reply:
I will remove, it was only to try see the envs in Jenkins